docs/workforce-management-framework/topics/con-extending-the-user-interface.adoc

```diff
@@ -0,0 +1,57 @@
+[id='{context}-con-extending-the-user-interface']
+= Extending the User Interface
```

Comment:
@wtrocki - "Extending the User Interface" - is this ok or am I missing a more descriptive heading?

Reply:
To provide more context. We have now only way to extend workorders data, but I wanted to create general space where all UI extensions will land in the future. Description is ok. 
If we want to extend that we can add `... angular user interface`